src/components/ListItem.tsx

```diff
@@ -142,14 +141,14 @@ export function ListItemCheckBox({ item, listPath }: Props) {
 						aria-checked={isChecked}
 						disabled={isChecked}
 					/>
-					<h4 className="text-nowrap me-1 mb-0"> {item.name} </h4>
+					<Form.Label className="">{item.name}</Form.Label>
```

Comment:
When I tested it, the larger text size pushes some content out. I'll use a responsive font size when I do merge!

https://github.com/user-attachments/assets/83c55d7e-45d6-488d-ab80-f6cee77f8b5c



.item-text {
		font-size: 0.75em;
	}
}